docs/keto/guides/userset-rewrites.mdx

```diff
@@ -1,12 +1,11 @@
-id: userset-rewrites title: Define file access permissions with userset rewrites sidebar_label: Userset rewrites
-
+---
+id: userset-rewrites
+title: Define global permission rules using OPL
```

Comment:
```suggestion
title: Define global permission rules using Ory Permission Language
```

I think it's okay to have a longer title and include the full name instead of just the abbreviation. This should also help with the searchability of the docs.